game.js

```diff
@@ -328,6 +365,25 @@ class Game extends Screen {
 		}
 	}
 
+	returntoMenuAfterWin() {
+			
+			this.noGuessChar = '_'
```

Comment:
This 'reset' code (the same as in the constructor should really go in the `init` function which gets called when the game starts (from the main menu).  Then we don't have to repeat things.